app/plugin_api.go

```diff
@@ -330,6 +330,10 @@ func (api *PluginAPI) GetChannel(channelId string) (*model.Channel, *model.AppEr
 	return api.app.GetChannel(channelId)
 }
 
+func (api *PluginAPI) GetChannels(options *model.GetChannelsOptions) (*model.ChannelList, *model.AppError) {
```

Comment:
Given that `options` is a required parameter, it should be a struct, not a pointer.